File: src/main/java/com/mathieuaime/hhf/bar/repository/BarRepository.java
package com.mathieuaime.hhf.bar.repository;

import com.mathieuaime.hhf.bar.model.BarDocument;
import org.springframework.data.repository.reactive.ReactiveCrudRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface BarRepository extends ReactiveCrudRepository<BarDocument, String> {

}

File: src/main/kotlin/com/mathieuaime/hhf/core/bar/repository/BarRepository.kt
package com.mathieuaime.hhf.core.bar.repository

import com.mathieuaime.hhf.core.bar.model.BarEntity
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface BarRepository : JpaRepository<BarEntity, String>